src/server/views/dashboard/lists-edit-annual-review-date.njk

```diff
@@ -0,0 +1,68 @@
+{% from "button/macro.njk" import govukButton %}
+{% extends "./dashboard.njk" %}
+
+{% block aside %}
+{% endblock %}
+
+{% block dashboard %}
+
+  {% if error.text %}
+    {{ dashboardMacros.errorBanner(error) }}
+  {% endif %}
+
+  <form class="form" action="{{ dashboardRoutes.listsEditAnnualReviewDate.replace(':listId', list.id) }}" method="post" required>
+    <input type="hidden" name="_csrf" id="_csrf" value="{{ csrfToken }}" />
+
+    <h2 class="govuk-heading-m" style="color: #b1b4b6;">{{list.country.name}}</h2>
+
+    <h1 class="govuk-heading-l">
+      Change annual review start date
+    </h1>
+
+    <p class="govuk-body">The next funeral directors list annual review starts <strong>{{ annualReviewStartDate }}.</strong></p>
+
+    <div class="govuk-form-group {{ "govuk-form-group--error" if error.text }}" id="annualReveiwDateForm">
+        <h2 class="govuk-heading-m govuk-!-margin-0">Enter a new start date</h2>
+        <fieldset class="govuk-fieldset" role="group" aria-describedby="annual-review-date">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--l">
+          </legend>
+          <div id="annual-review-date" class="govuk-hint">
+              The new date must be within 6 months of today’s date
```

Comment:
agreed.  @RichardBray can you check with Neil on this one?

Reply:
I can ask but I've put what I saw in the designs so it might not change 🤷🏿 

https://www.figma.com/file/u6oF0K1KU8jZhBEgwjfFiT/Annual-review-filters%2C-tags%2C-settings-wireframes-27-Sep-22?node-id=0%3A1

<img width="561" alt="Screenshot 2022-11-07 at 16 29 20" src="https://user-images.githubusercontent.com/1377253/200363094-3150c49d-0123-48e9-bda2-b3ccf8654394.png">

Reply:
as mentioned above, the logic used to get the next AR start date needs to be the same as used when validating it after submitting the POST req